### PR TITLE
Change lc0_main.go default server to main

### PR DESCRIPTION
**Pull request by Tilps**
Also add a flag to easily point it back at test server if desired.

Do Not Submit until we're actually ready for the switch over.

### CODE REVIEW

**Comment by dubslow** [comment]
I'm very, very skeptical about hard coding IP addresses. It should basically never be done that way, at least for something as high level as this IMO.

**Comment by Tilps** [comment]
Thats fine - but to be clear this is a copy/paste from main.go :P

**Comment by dubslow** [comment]
gross

File: lc0_main.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ var (
 	totalGames int
 	pendingNextGame *client.NextGameResponse
 
-	hostname = flag.String("hostname", "http://testserver.lczero.org", "Address of the server")
+	hostname = flag.String("hostname", "http://162.217.248.187", "Address of the server")
```
Comment (mooskagh) [inline lc0_main.go]: There is `http://api.lczero.org/` address for that!
Let's use it

Comment (Tilps) [reply lc0_main.go]: Done!

**Pull request merged**